test/mobile/nnc/test_aot_compile.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+TORCH_INSTALL_DIR=$(python -c "import site; print(site.getsitepackages()[0])")/torch
+TORCH_BIN_DIR="$TORCH_INSTALL_DIR"/bin
+CURRENT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+MODEL=aot_test_model.pt
+COMPILED_MODEL=aot_test_model.compiled.pt
+COMPILED_CODE=aot_test_model.compiled.ll
+
+test_aot_model_compiler() {
+  python "$CURRENT_DIR"/aot_test_model.py
+  exit_code=$?
```

Comment:
I think if you set "set -e -o pipefail" at the beginning of the shell script, it will automatically fail if any subshell it launches fails. You don't have to check it here explicitly.